src/tools/fuzzing/fuzzing.cpp

```diff
@@ -3445,7 +3446,6 @@ Expression* TranslateToFuzzReader::makeBasicRef(Type type) {
       // TODO: support actual non-nullable externrefs via imported globals or
       // similar.
       if (!type.isNullable()) {
-        assert(funcContext);
```

Comment:
It might be nice to add a comment about why we can't assert we're in a funcContext here.